chsdi/tests/mapproxy/test_wmtscapabilities.py

```diff
@@ -0,0 +1,112 @@
+#-*- coding: utf-8 -*-
+
+import sys
+import os
+import random
+from unittest import TestCase
+from testconfig import config as tc
+
+
+BAD_REFERER = 'http://foo.ch'
+
+GOOD_REFERER = 'http://unittest.geo.admin.ch'
+
+EPSGS = [21781, 4326, 4258, 2056, 3857]
+
+
+TILES_IDX = {3857: [(7, 67, 45), (10, 533, 360)],
+             21781: [(17, 5, 6), (18, 11, 13)],
+             2056: [(17, 5, 6), (18, 11, 13)],
+             4326: [(10, 6, 4), (15, 222, 123)],
+             4258: [(10, 6, 4), (15, 222, 123)]
+             }
+
+
+class TestWmtsCapabilitiesView(TestCase):
```

Comment:
I think this could be moved into the 'integration' tests (https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/tests/integration/test_wmtscapabilities.py#L6) as we don't test anything varnish related. It's fast as well, so it can be run on standard nose tests.